engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -366,6 +367,16 @@ object Main {
       )
       .build()
 
+    val warningsLimitOption = CliOption.builder
+      .longOpt(WARNINGS_LIMIT)
+      .hasArg(true)
+      .numberOfArgs(1)
+      .argName("limit")
+      .desc(
+        "Specifies a maximal number of reported warnings. Defaults to `100`."
```

Comment:
This number appears in 3 different places across the project.

I'm wondering when it will get out of sync 😅 
Do you think we could have some common constant that could be included in all places?

Reply:
It's tricky, not only with this option. We don't want to share the dependencies between projects for that.